internal/target/repository.go

```diff
@@ -576,24 +595,24 @@ func (r *Repository) SetTargetHostSets(ctx context.Context, targetId string, tar
 			msgs := make([]*oplog.Message, 0, 2)
 			targetTicket, err := w.GetTicket(target)
 			if err != nil {
-				return fmt.Errorf("set target host sets: unable to get ticket: %w", err)
+				return errors.Wrap(err, op)
```

Comment:
Other ticket errors included:  errors.WithMsg("unable to get ticket")

Reply:
Oh good catch thanks!